src/pal/inc/pal.h

```diff
@@ -2411,12 +2411,10 @@ TlsFree(
 
 PALIMPORT
 void *
```

Comment:
Should these be rather changed to return `PVOID` for consistency with all other PAL APIs and stay ``PALIMPORT` + `PALAPI` ?

Reply:
Fixed and changed all the (non-`const`) `void *`s in `pal.h` with `PVOID`.